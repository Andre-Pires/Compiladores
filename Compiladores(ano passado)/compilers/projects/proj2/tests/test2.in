float f(int a){
  int x;
  x=x + a * (int)3.4;
  while(3.5){
    x=3.78;
  };
}
