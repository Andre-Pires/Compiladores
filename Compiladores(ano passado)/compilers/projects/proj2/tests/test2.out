type error: int expression expected, line 4
